exercises/practice/square-root/.docs/instructions.md

```diff
@@ -0,0 +1,9 @@
+# Description
+
+Given a natural radicand, return its square root.
+
+Note that the term "radicand" refers to the number for which the root is to be determined. That is, it is the number under the root symbol.
```

Comment:
We may have to add that this is a simplified exercise where only perfect squares are tested.

This way, we may avoid questions where the student asks what happens if functions takes, for example, '3'.

We may also add an anylizer ensuring that `:math.sqrt(x)` is not used. Note that maybe just a hint is ok here, what do you think @angelikatyborska

Reply:
Yes, we definitely need to tell students not to use `:math.sqrt()`. I would recommend:
1. Adding an `instructions.append.md` file to tell students that they need to implement an algorithm on their own and not use `:math.sqrt()` or `Float.pow`
2. Adding an issue to the analyzer repo so that somebody works on it at some point, an issue similar to https://github.com/exercism/elixir-analyzer/issues/42 for example

Reply:
> We may have to add that this is a simplified exercise where only perfect squares are tested.

That will be apparent looking at the tests, I think. It's not uncommon for tests to have more information than the description :)